recipe/meta.yaml

```diff
@@ -12,35 +12,42 @@ source:
   sha256: e8e1017b2cf1dda767aef19d2fd9ee5ebe07e050d430f77a0a7c66ba08b8cdae
 
 build:
-  number: 1
-  noarch: python
+  number: 0
   script:
-    # We are currently building a noarch package, but pysoundfile contains a
-    # install-time check for the OS and injects the sf_wchar_open function
-    # conditionally. If this variable is not set, opening files on Windows doesn't work
-    # Fix coming hopefully with https://github.com/bastibe/python-soundfile/pull/288
-    # Original issue on conda-forge: https://github.com/conda-forge/libsndfile-feedstock/issues/14
-    - export PYSOUNDFILE_PLATFORM=win32
-    - python -m pip install . --no-deps -vv
+      # To make sure we really do not bring in _soundfile_data
+    - rm -rf _soundfile_data
```

Comment:
This will fail on Windows

Reply:
Since it's being skipped on windows anyways no need to add anything correct?

Reply:
I think it's fine personally

Reply:
in the tarball there is no such folder, removed also for not win